src/mono/wasm/Wasm.Build.Tests/PInvokeTableGeneratorTests.cs

```diff
@@ -884,5 +884,80 @@ public static unsafe int Main(string[] argv)
         [BuildAndRun(host: RunHost.Chrome, aot: false)]
         public void EnsureWasmAbiRulesAreFollowedInInterpreter(BuildArgs buildArgs, RunHost host, string id) =>
             EnsureWasmAbiRulesAreFollowed(buildArgs, host, id);
+
+        [Theory]
+        [BuildAndRun(host: RunHost.Chrome, aot: true)]
+        public void UCOWithSpecialCharacters(BuildArgs buildArgs, RunHost host, string id)
+        {
+            string code =
```

Comment:
We typically try to store the WBT program code in `https://github.com/dotnet/runtime/tree/main/src/mono/wasm/testassets/Wasm.Buid.Tests.Programs`. It's easier to read the code if it's not enclosed in the quotation, it's reusable in other tests and can be run separately if needed, without copy-paste.

Reply:
I see, but this block will not be reused (at least for now), do you think it is better to store it ?

Reply:
Yes. There is still the readability and standalone manual runs argument. Actually when I check, currently we use each file from that directory only once.